cmd/launch_producer/main.cpp

```diff
@@ -60,9 +62,9 @@ typedef void (*FN_PTR)(void);
 
 const char* kDriverProperty = "ro.gfx.driver.1";
 const char* kProducerPaths[] = {
-    "/assets/libgpudataproducer.so",
     "/lib/" ABI "/libgpudataproducer.so",
 };
+const char* kPidFileName = "/data/local/tmp/launch_producer.pid";
```

Comment:
As anyone can read/write to `/data/local/tmp`, I would prefer to prefix this name with gapid:
`/data/local/tmp/gapid_launch_producer`